Extension/src/common.ts

```diff
@@ -544,19 +544,35 @@ export async function checkInstallBinariesExist(): Promise<boolean> {
 export async function checkInstallJsonsExist(): Promise<boolean> {
```

Comment:
I was looking at this case -- looks like it should also be changed to /**, but that's unrelated to your changes so you don't need to fix that -- we probably have other cases of `/*` that should be fixed. I never realized it was an issue.